### PR TITLE
Updated write-applet.xml for Cinnamon 4.6 to 5.8+

### PR DESCRIPTION
**Pull request by claudiux**
Changing example.

### CODE REVIEW

File: docs/reference/cinnamon-tutorials/write-applet.xml [hunk 1]
```diff
@@ -90,28 +95,36 @@
...
-                this.set_applet_tooltip(_("Click here to kill a window"));
-            },
-
-            on_applet_clicked: function() {
-                Util.spawn(['xkill']);
+            on_applet_clicked(event) {
+                // Please consult 'man notify-send'.
+                let command = `bash -c 'notify-send -u normal `;
+                command += `"Thank you ${user}" "for improving Cinnamon!"'`;
+                Util.spawnCommandLineAsync(command)
```
Comment (mtwebster) [inline docs/reference/cinnamon-tutorials/write-applet.xml:122]: Maybe a comment emphasizing that async should always try to be used, and why.

File: docs/reference/cinnamon-tutorials/write-applet.xml [hunk 2]
```diff
@@ -90,28 +95,36 @@
       <programlisting>
         const Applet = imports.ui.applet;
         const Util = imports.misc.util;
+        const GLib = imports.gi.GLib;
+
+        /**
+         * Variables (and other constants)
+         */
+        var user = GLib.getenv("USER"); // acts like the shell command: echo -n $USER
```
Comment (mtwebster) [inline docs/reference/cinnamon-tutorials/write-applet.xml:103]: Maybe GLib.get_user_name()?

File: docs/reference/cinnamon-tutorials/write-applet.xml [hunk 3]
```diff
@@ -90,28 +95,36 @@
...
+
+        user = user.charAt(0).toUpperCase() + user.slice(1); // Capitalize 'user'
```
Comment (mtwebster) [inline docs/reference/cinnamon-tutorials/write-applet.xml:105]: we also have String.prototype.capitalize() (overrides.js)

File: docs/reference/cinnamon-tutorials/write-applet.xml (hunk 1 above)
Comment (claudiux) [reply docs/reference/cinnamon-tutorials/write-applet.xml:122]: Done!

**Pull request merged**